src/Bridges/Constraint/geomean_to_relentr.jl

```diff
@@ -0,0 +1,95 @@
+"""
+    GeoMeantoRelEntrBridge{T}
+
+The `geometric mean cone` is representable with a relative entropy constraint and a nonnegative auxiliary variable,
+since ``u \\le \\prod_{i=1}^n w_i^{1/n}`` if and only if there exists a ``y \\ge 0`` such that
+``0 \\ge \\sum_{i=1}^n (u + y) \\log (\\frac{u + y}{w_i})``, or equivalently
```

Comment:
Can you detail a bit more where this reduction come from or give a ref ?

Reply:
I didn't see this anywhere. But I can try to show why the math works

Reply:
See updated explanation